test/unit_tests/datautil/test_transforms.py

```diff
@@ -71,3 +70,19 @@ def test_transform_windows_method(windows_concat_ds):
     raw_window = windows_concat_ds[0][0]
     transform_concat_ds(windows_concat_ds, transforms)
     assert not np.array_equal(raw_window, windows_concat_ds[0][0])
+
+
+def test_zscore_base_concat_ds(base_concat_ds):
```

Comment:
If `zscore` is changed above to also accept windowed data, we should add another test here for the windowed case.